csharp/ql/src/utils/modeleditor/FetchApplicationModeMethods.ql

```diff
@@ -0,0 +1,32 @@
+/**
+ * @name Fetch model editor methods (application mode)
```

Comment:
nit: Maybe consider to avoid the use of the word "methods" or elaborate a bit "methods and other callables".